samples/local_attestation/host/host.cpp

```diff
@@ -5,6 +5,7 @@
 #include <stdio.h>
 #include "localattestation_u.h"
 
+
```

Comment:
Remove this newly added empty line and two others below.

Reply:
Fixed.